src/components/VmStatus/getVmStatus.js

```diff
@@ -156,17 +126,132 @@ const isWaitingForVmi = vm => {
   return NOT_HANDLED;
 };
 
-export const getVmStatusDetail = (vm, launcherPod, importerPods, migration) =>
-  isBeingMigrated(vm, migration) || // must be precceding isRunning() since vm.status.ready is true for a migrating VM
+const findCdiPod = (volume, cdiPods = []) =>
```

Comment:
wouldn't it be better to move this method to some util file?

Reply:
moved to `utils/selectors`